src/test/kotlin/tech/harmonysoft/oss/gradle/dist/CustomGradleDistributionPluginTest.kt

```diff
@@ -521,17 +570,24 @@ class CustomGradleDistributionPluginTest {
         return result
     }
 
-    private fun runAndVerify(testFiles: TestFiles) {
+    private fun runAndVerify(testFiles: TestFiles, buildDistsDir: String = BUILT_DISTS_DIR) {
         GradleRunner.create()
             .withProjectDir(testFiles.inputRootDir)
             .withArguments("buildGradleDist", "--stacktrace")
             .withPluginClasspath()
             .withDebug(true)
             .build()
 
-        verify(testFiles.expectedRootDir, File(testFiles.inputRootDir, "build/gradle-dist"))
+        verify(testFiles.expectedRootDir, File(testFiles.inputRootDir, buildDistsDir))
+    }
+
+    private fun withDistributionType(type: String): CustomGradleDistributionPluginTest {
```

Comment:
we can just create the both dummy `-bin.zip` and `-all.zip` in tests and not bother with explicitly specifying the target type. Less code - easier to maintain

Reply:
At your request, I just simplified it :)